Meadowlark-js/.env.example

```diff
@@ -4,6 +4,9 @@ POSTGRES_USER=<postgres user>
 POSTGRES_PASSWORD=<postgres password>
 
 # OAuth 2.0 signing key: base 64 encoded, 256 bit key. To create a new one,
-# try running the application and accessing endpoint /{stage}/createSigningKey
+# try running the application and accessing endpoint /{stage}/oauth/createSigningKey
 # or run `openssl rand -base64 256`.
-SIGNING_KEY=<signing key>
+OAUTH_SIGNING_KEY=<signing key>
+
+OAUTH_SERVER_ENDPOINT_FOR_OWN_TOKEN_REQUEST=http://localhost:3000/local/api/oauth/token
+OAUTH_SERVER_ENDPOINT_FOR_TOKEN_VERIFICATION=http://localhost:3000/local/api/oauth/verify
```

Comment:
Shouldn't the `api` be removed?

Reply:
Wonder if I missed a push, thought I did that...

Reply:
I foolishly did a commit amend on that fix last night and didn't force push. Don't want to force push right now and mess up the commits you have already looked at. But github is not letting me commit the two code suggestions I have below - I add to batch and then hit the commit button, and nothing happens. @bradbanister @andonyns can one of you try committing the two suggested code updates, right here in GitHub?